client.go

```diff
@@ -461,10 +457,13 @@ func (c *Client) runPublisher(ouputChan *amqp.Channel) {
 
 				c.retryRequest(request, err)
 
-				c.errorLog(
-					"client: publisher stopped because of error: %s, request: %s",
-					err.Error(),
-					stringifyRequestForLog(request),
+				c.logger.Error(
+					"client: publisher stopped because of error",
+					slog.Any("error", err),
+					slog.Attr{
+						Key:   "request",
+						Value: slog.GroupValue(logAttrsForRequest(request)...),
+					},
```

Comment:
This seems somewhat repeated but not sure it's worth adding a method. Just thinking it would read nice with something like

```go
func slogGroupFor(key string, attrs: []slog.Attr) slog.Attr {
	return slog.Attr{
		Key:   key,
		Value: slog.GroupValue(attrs...),
	}
}

func (c *Client) fn() {
	c.logger.Error(
		"something went wrong",
		slog.Any("error", err),
		slogGroupFor("request", logAttrsForRequest(request))
	)
}
```

It could be re-used for every `request`, `response`, `table`, `delivery` etc.